hermes-api/src/main/java/pl/allegro/tech/hermes/api/TopicNameWithMetrics.java

```diff
@@ -24,7 +24,7 @@ public TopicNameWithMetrics(
             @JsonProperty("subscriptions") int subscriptions,
             @JsonProperty("throughput") MetricDecimalValue throughput,
             @JsonProperty("volume") long volume,
-            @JsonProperty("qualifiedName") String qualifiedName
+            @JsonProperty("name") String qualifiedName
```

Comment:
Changed so that the attribute name is equal to that present in Topic class. This allows filtering by name for both classes.